models/illiad_client.rb

```diff
@@ -0,0 +1,36 @@
+require 'httparty'
+
+class ILLiadClient
+  include HTTParty
+  base_uri "#{ENV.fetch('ILLIAD_API_HOST')}/webplatform"
+
+  def initialize()
+    self.class.headers 'ApiKey' => ENV.fetch('ILLIAD_API_KEY')
+    self.class.headers 'Accept' => 'application/json'
+  end
+
+  def get(url, query={})
+    self.class.get(url, query: query)
+  end
+
+  def get_all(url:, record_key:, limit: 100, query: {})
```

Comment:
I think we should get rid of get_all because this logic is specific to the Alma API

Reply:
`get_all` has been removed.